packages/configure-splash-screen/src/cli-command.ts

```diff
@@ -0,0 +1,196 @@
+import { Command } from 'commander';
+import path from 'path';
+
+import configureAndroid from './android';
+import configureIos from './ios';
+import {
+  SplashScreenImageResizeMode,
+  SplashScreenStatusBarStyle,
+  Platform,
+  PlatformType,
+} from './constants';
+import { AndroidSplashScreenJsonConfig, IosSplashScreenJsonConfig } from './types';
+import { validateEnumValue } from './validators';
+
+const AVAILABLE_OPTIONS_NAMES = [
+  'imageResizeMode',
+  'platform',
+  'backgroundColor',
+  'imagePath',
+  'darkModeBackgroundColor',
+  'darkModeImagePath',
+  'statusBarHidden',
+  'statusBarStyle',
+  'darkModeStatusBarStyle',
+  'statusBarTranslucent',
+  'statusBarBackgroundColor',
+  'darkModeStatusBarBackgroundColor',
+] as const;
+
+type CLIOptionName = typeof AVAILABLE_OPTIONS_NAMES[number];
```

Comment:
Why this sophisticated `const AVAILABLE_OPTIONS_NAMES` and then this `typeof` instead of a simple enum or a union type?